### PR TITLE
Create "Becoming a remote-first company by practicing openness" chapter

### PR DESCRIPTION
**Pull request by MaineC**
This relates to #25. It's a first draft of how combining InnerSource at the engineering level with Open Organisation practices for higher level processes helps companies successfully move towards a remote-friendly, if not remote-first working model. It is based on experience from a German FinTech that was founded 20 years ago.

### CODE REVIEW

**Review by semioticrobotic — APPROVED** [review]
Looks great, @MaineC. I'm so happy to be able to include this in the guide.

File: 00_drostfromm_remote-first-through-openess.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+Our idea was to build a transparent process, one that's open to everyone and focused on those who have a stake in the decision at hand.
+Here's what we came up with:
+
+1. First, a problem in need of a common, cross unit/ cross team solution should be described in writing in an issue tracker.
+In that issue, everyone is invited to participate.
+The goal is to reach a shared understanding of the problem and sketch possible solutions together.
+2. Assuming that participants were able to arrive at an agreeable solution during that discussion, the team should then write a formal proposal and post it as a pull request.
+The goal here is to let people participate in the solution design process.
+Some key aspects of this phase are:
+   1. Explicitly inviting participation from potential stakeholders are not yet aware of the discussion
```
Comment (mishari) [inline 00_drostfromm_remote-first-through-openess.md:148]: "stakeholders **who** are not yet aware"

File: 00_drostfromm_remote-first-through-openess.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+   2. Having all relevant communication tracked, so people who join the discussion later can follow what was already discussed
+   3. If the case of competing alternative solutions, documenting why those alternatives were not implemented or pursued
```
Comment (mishari) [inline 00_drostfromm_remote-first-through-openess.md:150]: **In** the case of competing alternative

File: 00_drostfromm_remote-first-through-openess.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+   4. Describing what the solution should cover—its advantages and implications, and a description of how the solution addresses the original problem (not to mention the limits of the solution)
+   5. To make communication clearer (and in cases where the goal is to create a cross-team standard for solving a specific technical problem), using specific words and phrases endorsed by the Internet Engineering Task Force when specifying characteristics
+3. If the proposal meets no veto (and any veto must be accompanied by a detailed explanation), and if at least two senior contributors explicitly vote in favor of the proposed solution, it can be counted accepted, is merged, and gets promoted across all teams affected going forward.
```
Comment (mishari) [inline 00_drostfromm_remote-first-through-openess.md:153]: If the proposal does not meet a veto
counted **as** accepted

**Review by mishari — COMMENTED** [review]
Thank you! I just proposed a few minor edits.

File: 00_drostfromm_remote-first-through-openess.md (hunk 1 above)
Comment (semioticrobotic) [reply 00_drostfromm_remote-first-through-openess.md:148]: Got it. Thanks!

File: 00_drostfromm_remote-first-through-openess.md (hunk 2 above)
Comment (semioticrobotic) [reply 00_drostfromm_remote-first-through-openess.md:150]: Great catch.

File: 00_drostfromm_remote-first-through-openess.md (hunk 3 above)
Comment (semioticrobotic) [reply 00_drostfromm_remote-first-through-openess.md:153]: Got it!

**Comment by semioticrobotic** [comment]
Thanks so much, @mishari, for these great catches. Since we've already merged this work into the main branch, I'll be incorporating them all there.

**Pull request merged**